panphon/permissive.py

```diff
@@ -134,10 +134,10 @@ def fts_match(self, fts_mask, segment):
             bool: None if `segment` cannot be parsed; True if the feature values
                   of `fts_mask` are a subset of those for `segment`
         """
-        fts_mask = set(fts_mask)
         fts_seg = self.fts(segment)
         if fts_seg:
-            return fts_seg <= fts_mask
+            fts_mask = set(fts_mask)
+            return fts_mask <= segment
```

Comment:
Note how this is inverted from before but now matches the condition in the docstring and in _panphon:205

Reply:
This appears to be asking whether a feature mask matches a Unicode segment. The original code is wrong, though. It should be:
```python
if fts_seg:
   return fts_mask <= fts_seg
```

Reply:
fixed now.